conf/cassandra.yaml

```diff
@@ -1805,3 +1805,10 @@ drop_compact_storage_enabled: false
 #    heartbeat_file: /var/lib/cassandra/data/cassandra-heartbeat
 #    excluded_keyspaces: # comma separated list of keyspaces to exclude from the check
 #    excluded_tables: # comma separated list of keyspace.table pairs to exclude from the check
+
+sstable_compressor:
+#  Defines the compression params used to read/write the sstables.  If undefined the old default is used.
+  - class_name: org.apache.cassandra.io.compress.LZ4Compressor # the comnpressor class
+    parameters:
+      - chunk_length_in_kb: '16384'
```

Comment:
that's 16MBs! 
and not how we do unit type config settings anymore…

Reply:
not sure about the how. just that we would rather not have untyped settings in yaml now

Reply:
i suggest a separate class. CompressionParms is used for inside the schema.